resources/assets/js/annotations/annotatorContainer.vue

```diff
@@ -435,8 +435,19 @@ export default {
                     // TODO: confidence control
                     confidence: 1,
                 };
+
+                if (this.isVolareAnnotationMode) {
+                    this.$refs.canvas.setEditingStyle(annotation);
+                }
+
```

Comment:
I'd prefer this to have a fixed "blink rate", i.e. when the label is attached, the annotation blinks with the same speed independent from the API request. You can do this here with something like:
```suggestion

                if (this.isVolareAnnotationMode) {
                    this.$refs.canvas.blinkAnnotation(annotation);
                }
```
Then there is no need for the code change below. If there is an error with the API request, it will be handled by `handleErrorResponse`.

For the blinking animation maybe apply the editing style for 250 ms?